src/screens/GroupSetting/index.js

```diff
@@ -67,14 +66,14 @@ const GroupSetting = ({navigation, route}) => {
         <View style={styles.users}>
           <Text style={styles.countUser}>{`Participants (${countUser})`}</Text>
           <FlatList
-            data={Object.keys(participants)}
+            data={participants}
             keyExtractor={(item, index) => index.toString()}
             renderItem={({item}) => (
               <View style={{height: 72}}>
                 <ProfileContact
                   key={item}
-                  fullname={getProfileName(participants[item].user.name)}
-                  photo={participants[item].user.image}
+                  fullname={getProfileName(item?.user?.name)}
+                  photo={item?.user?.image}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

There is a potential issue with the `keyExtractor` function in the `FlatList` component. Using the index as a key is not recommended because it doesn't uniquely identify the item, especially when the list can change. This can lead to unexpected behavior in your UI.

Instead, you should use a unique property from each item for the key. If the `item` has an `id` or similar unique field, that would be a better choice. Here's how you could modify the `keyExtractor`:

```diff
- keyExtractor={(item, index) => index.toString()}
+ keyExtractor={(item) => item.id}
```

Also, there seems to be a change in the data structure of `participants`. Please ensure that the new structure is correctly handled everywhere this data is used. Specifically, check if `item?.user?.name` and `item?.user?.image` are valid paths in the new data structure.


<!-- This is an auto-generated comment by OSS CodeRabbit -->